### DISCUSSION TITLE
Empty Action Wheel

### DISCUSSION THREAD

**Discussion by purplelacquer** [ErinaSugino/Starbound-SBR-Mod#101] [Trouble Shooting] [answered]
Same problems as another bug report, sex nodes activate with an empty wheel and leaving the scene leaves a copy of my character still there. Sex between NPC's works fine however.

[starbound.log](https://github.com/ErinaSugino/Starbound-SBR-Mod/files/15317338/starbound.log)

**Comment by ErinaSugino** [comment, marked as answer]
The issue in question is your character wearing a weirdly invalid item - something to do with invisible clothing defined on the penguin race?
I remember hearing about that one in the past, but I can't find any information on it anymore, so I also cannot tell you which mod is causing this. But generally, removing any invisible clothing on you should fic it.

**Reply by purplelacquer** [reply]
Can't believe that didn't occur to me lol, removing the set fixes it! I also took the time to check my modlist and I think the clothes are added by Betabound. Could be wrong, but thanks for the help :)
